### PR TITLE
Adding device selection support 

### PR DESCRIPTION
**Pull request by Virgile-Dauge**
Thx for your node, it's quite usefull to skip ZED-SDK (no GPU on my laptop).

Unfortunately, I wasn't able to use your node as it is, due to my laptop webcam taking the first video slot `/dev/video0`. 

The way to fix this issue was to change the device parameter here : `camera_ = new cv::VideoCapture(0);` to `camera_ = new cv::VideoCapture(1);`. 

It was quite easy to make it a launching argument for the node, so i did it :). 

Hope it will be helpfull !

### CODE REVIEW

File: src/zed_cpu_ros.cpp [hunk 1]
```diff
@@ -31,9 +31,10 @@ class StereoCamera
 	 * @param[in]  resolution  The resolution
 	 * @param[in]  frame_rate  The frame rate
 	 */
-	StereoCamera(int resolution, double frame_rate): frame_rate_(30.0) {
+	 StereoCamera(int device, int resolution, double frame_rate)
+ : frame_rate_(30.0) {
```
Comment (willdzeng) [inline src/zed_cpu_ros.cpp]: can you make this on the same line? Thanks

File: launch/zed_cpu_ros.launch [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <launch>
 <arg name="config_file_location" default="$(find zed_cpu_ros)/config/SN1267.conf"/>
 <arg name="camera_namespace" default="camera"/>
+<arg name="device" default="0"/>
```
Comment (willdzeng) [inline launch/zed_cpu_ros.launch]: change to device_id every where

File: README.md [hunk 3]
```diff
@@ -53,18 +53,19 @@ A simple zed camera driver which only use CPU and only publish left and right ra
...
- _                            | _                                                           | '3': VGA                                    
- frame_rate                   | Rate at which images are published                          | int                                             
- left_frame_id                | Left Frame ID                                               | string        
- right_frame_id               | Right Frame ID                                              | string        
- load_zed_config              | Whether to use ZED calibration file                         | bool        
- config_file_location         | The location of ZED calibration file                        | string        
- show_image                   | Whether to use opencv show image                            | bool        
+ Parameter                    |           Description                                       |              Value
+------------------------------|-------------------------------------------------------------|-------------------------
+ device                   | device ID selection (if multiple cameras)                                      | int
```
Comment (willdzeng) [inline README.md]: fix the format?

**Review by willdzeng — CHANGES_REQUESTED** [review]
Thanks for fixing this, just a few comments need to be addressed

File: README.md (hunk 3 above)
Comment (Virgile-Dauge) [reply README.md]: I was using auto-formatting tool (Atom Beautify) which changed indentation in several files. I just reverse patched those changes.

File: launch/zed_cpu_ros.launch (hunk 2 above)
Comment (Virgile-Dauge) [reply launch/zed_cpu_ros.launch]: Done in 311cb52 :)

File: src/zed_cpu_ros.cpp (hunk 1 above)
Comment (Virgile-Dauge) [reply src/zed_cpu_ros.cpp]: Done in 311cb52 :)

**Review by willdzeng — APPROVED** [review]

**Comment by willdzeng** [comment]
Thank you for your contribution!!

**Pull request merged**